solr/core/src/java/org/apache/solr/handler/admin/CoreAdminHandler.java

```diff
@@ -190,7 +212,7 @@ public void handleRequestBody(SolrQueryRequest req, SolrQueryResponse rsp) throw
 
       // Pick the action
       final String action = req.getParams().get(ACTION, STATUS.toString()).toLowerCase(Locale.ROOT);
-      CoreAdminOperation op = opMap.get(action);
+      CoreAdminOp op = opMap.get(action);
       if (op == null) {
         handleCustomAction(req, rsp);
```

Comment:
minor: maybe we could INFO log the `action` here i.e. the `@Deprecated` annotation on `handleCustomAction` method shows that custom function might be used and operationally logging here would show that/when/how the custom function is used
```suggestion
        log.info("action '{}' not found, calling custom action handler", action);
        handleCustomAction(req, rsp);
```

Reply:
Sure, no problem! We can give some hint like saying "if you target some custom behaviour use custom actions instead"

Reply:
I think maybe using message with `warn` severity would be better here? I added following at the moment:

```java
      log.warn("action '{}' not found, calling custom action handler. "
          + "If original intention was to target some custom behaviour "
          + "use custom actions defined in 'solr.xml' instead", action);
```

Feel free to apply any changes